docs/projects/version-control/index.md

```diff
@@ -5,68 +5,71 @@ position: 110
 
-## Configuring a project to be version-controlled 
+This update covers the ability to version control text variables in your Project and also manage references to the following variable types within Git:
 
-Version-control is configured per project and is accessed via the **{{Settings,Version Control}}** navigation menu item. 
+* Certificate
+* Account
+* Worker Pool
+* Prompted
 
-Learn more about [Configuring version control on a project](/docs/projects/version-control/converting/index.md).
+You get the same control as the Octopus UI allowing you to scope your variables to environments, roles, specific targets and more without restrictions or adding any specific formatting or syntax. 
 
-## Config-as-code reference
+![Octopus Deploy branch selector](todo.png)
 
-Several resources previously stored in SQL Server will now be stored in git once a project is version-controlled.
+### Creating safe, reliable and auditable “As Code” deployment pipelines
 
-Learn more about [Configuration as Code reference](/docs/projects/version-control/config-as-code-reference.md)
+With version controlled deployment processes and variables, Octopus provides safe, reliable and auditable “As Code” deployment pipelines. This lets your teams work in line with your company's regulatory requirements and compliance policies, making auditing a much smoother process.  
 
-## Making changes to a version-controlled project
+- **Traceable end-to-end CI/CD “As Code” pipelines** - Octopus has built-in plugins for a wide range of CI servers for easy integration. You can pair Octopus with a CI server that supports build definitions as code, giving you complete traceability into changes to your software development process. Changes to your build and deployment pipelines can go through the same pull request review and approval process, improving the stability and reliability of your pipeline.
+- **Consistent processes and approvals** - Improve the quality of your code and meet compliance requirements by using pull request workflows to review and approve changes. Consistent processes contribute to quality releases, and you can enforce policies so that unapproved changes don't get merged.
+- **Traceable changesets** - Have clear visibility into changesets by using pull request diffs combined with detailed audit logging. Octopus keeps a complete audit trail of everything that happens, with detailed logs and filters. See who changed what and when to help troubleshoot release issues. Capture further details with source control commits and Git diffs showing you the exact changes made to processes and configuration settings.
+- **Disaster recovery** - Back-up and recover user pipelines in Git. If you need to, roll-back changes to recover more easily from release breaks.
```

Comment:
```suggestion
- **Traceable end-to-end CI/CD “As Code” pipelines** - Octopus has built-in plugins for a wide range of CI servers for easy integration. You can pair Octopus with a CI server that supports build definitions as code, giving you complete traceability into changes to your software development process. Changes to your build and deployment pipelines can go through the same pull request review and approval process, improving the stability and reliability of your pipeline.
- **Consistent processes and approvals** - Improve the quality of your code and meet compliance requirements by using pull request workflows to review and approve changes. Consistent processes contribute to quality releases, and you can enforce policies so that unapproved changes don't get merged.
- **Traceable changesets** - Have clear visibility into changesets using pull request diffs combined with detailed audit logging. Octopus keeps a complete audit trail of everything that happens, with detailed logs and filters. See who changed what and when to help troubleshoot release issues. Capture further details with source control commits and Git diffs showing the changes made to processes and configuration settings.
- **Disaster recovery** - Back up and recover user pipelines in Git. If you need to, roll-back changes to recover more easily from release breaks.
```